docs/docs/action-server/actions.mdx

```diff
@@ -308,7 +308,7 @@ and return both messages to the user.
 ## Special Action Types
 
 There are special action types that are automatically triggered under certain circumstances, namely [default actions](../concepts/default-actions.mdx)
-and [slot validation actions](../concepts/slot-validation-actions.mdx).
+and [slot validation actions](../nlu-based-assistants/slot-validation-actions.mdx).
```

Comment:
@dakshvar22 , we are still using slot validation actions in calm, aren't we? if you have custom logic for how to accept / reject a slot value

Reply:
Yes, slot validation actions are still used.

Reply:
Yes, but only regular actions can be used and we also expect these actions now to be named `validate_<slot_name>` (I made a note of that under Flows `collect_information` step) and the rasa-sdk helper actions only work with nlu-based assistants, so this change is accurate.